app/Cardano/Shell/Features/Logging.hs

```diff
@@ -48,13 +56,16 @@ data LoggingParameters = LoggingParameters
 -- the functions effects and constraining the user (programmer) of those function to use specific effects in them.
 -- https://github.com/input-output-hk/cardano-sl/blob/develop/util/src/Pos/Util/Log/LogSafe.hs
 data LoggingLayer = LoggingLayer
-    { llBasicTrace :: forall m . MonadIO m => Trace m Text
-    , llLogDebug   :: forall m a. (MonadIO m, Show a) => Trace m a  -> a -> m ()
-    , llLogInfo    :: forall m a. (MonadIO m, Show a) => Trace m a  -> a -> m ()
-    , llLogNotice  :: forall m a. (MonadIO m, Show a) => Trace m a  -> a -> m ()
-    , llLogWarning :: forall m a. (MonadIO m, Show a) => Trace m a  -> a -> m ()
-    , llLogError   :: forall m a. (MonadIO m, Show a) => Trace m a  -> a -> m ()
-    , llAppendName :: forall m a. (MonadIO m, Show a) => LoggerName -> Trace m a -> m (Trace m a)
+    { llBasicTrace      :: forall m . MonadIO m => Trace m Text
+    , llLogDebug        :: forall m a. (MonadIO m, Show a) => Trace m a  -> a -> m ()
+    , llLogInfo         :: forall m a. (MonadIO m, Show a) => Trace m a  -> a -> m ()
+    , llLogNotice       :: forall m a. (MonadIO m, Show a) => Trace m a  -> a -> m ()
+    , llLogWarning      :: forall m a. (MonadIO m, Show a) => Trace m a  -> a -> m ()
+    , llLogError        :: forall m a. (MonadIO m, Show a) => Trace m a  -> a -> m ()
+    , llAppendName      :: forall m a. (MonadIO m, Show a) => LoggerName -> Trace m a -> m (Trace m a)
+    , llConfiguration   :: Configuration
```

Comment:
Like I said, I don't think we need a `Configuration`.

Reply:
The code in `cardano-ledger` currently uses the `llConfiguration` field.